lib/Text/Haml.pm

```diff
@@ -333,8 +333,6 @@ sub parse {
         if ($line =~ m/^(?:$tag_start
             |$class_start
             |$id_start
-            |$attributes_start[^$attributes_start]
```

Comment:
@vti do you remember why you were matching on `attributes_start` here? I'm worried there was a real reason, and thus I need to fix this in a different way, but I can't think of valid haml where this would be necessary.